coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
Regarding removing `LogPrinterMixin`, IMO, its better to keep it. Again it provides flexibility to capture logs using custom logger if someone wants to in bears. Its easy to redirect logs to any other stream using it.